.github/workflows/docker-image-build.yml

```diff
@@ -0,0 +1,153 @@
+name: Build Images
+
+on:
+  push:
+    branches:
+    - master
+    - multi-arch
```

Comment:
Maybe we would remove this branch before merging into master.

Reply:
Yes, of course.